src/data-table/__tests__/data-table-columns.e2e.js

```diff
@@ -487,8 +487,7 @@ describe('data table columns', () => {
 
     await datepicker.click({clickCount: 3});
     await page.keyboard.press('Backspace');
-    await datepicker.type('07132014');
-    await datepicker.type('07132014');
+    await datepicker.type('07–13–2014 – 07–13–2014');
```

Comment:
Is it possible to maintain the hyphen auto fills? Please make double check that the default keyboard input behavior continues to work as before